### DISCUSSION TITLE
How to contribute additional language support?

### DISCUSSION THREAD

**Discussion by dmitriy-shmilo** [codersrank-org/repo_info_extractor#202] [General]
Hello. Ho do we contribute new language detection to this tool? Is it as simple as adding new extension mapping to languagedetection/analyzer.go? How is the new language going to appear on https://codersrank.io/?

**Comment by peti2001** [comment]
Hi @dmitriy-shmilo, 
This is a good example of how to add a new language: https://github.com/codersrank-org/repo_info_extractor/pull/200
